tests/unit_tests/test_time_series.py

```diff
@@ -371,12 +372,10 @@ def test_check_none_matnwb_resolution_pass():
         load_namespaces=True,
         driver="ros3",
     ) as io:
-        nwbfile = robust_s3_read(command=io.read)
-        time_series = robust_s3_read(
-            command=nwbfile.processing["video_files"]["video"].time_series.get,
-            command_args=["20170203_KIB_01_s1.1.h264"],
-        )
-    assert check_resolution(time_series) is None
+        nwbfile = io.read()
+        time_series = nwbfile.processing["video_files"]["video"]["20170203_KIB_01_s1.1.h264"]
+
+        assert check_resolution(time_series=time_series) is None
```

Comment:
This one will just be removed in a follow-up. Not too concerned about it now; has to do with old versions of packages